cli/src/ai/log-capture.ts

```
@@ -0,0 +1,76 @@
+import { mkdir, unlink, writeFile, appendFile } from 'node:fs/promises'
```

ESLint

[error] Expected "appendFile" to come before "writeFile".

(perfectionist/sort-named-imports)